prog/mutation.go

```diff
@@ -374,6 +374,30 @@ func (t *BufferType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []
 	case BufferText:
 		data := append([]byte{}, a.Data()...)
 		a.data = r.mutateText(t.Text, data)
+	case BufferCompressed:
+		data := a.Data()
+		if len(data) == 0 {
+			return
+		}
+		data, err := Decompress(data)
+		if err != nil || len(data) == 0 {
+			return // Leave data unchanged.
```

Comment:
There are 2 potential problems here:
1. If we add some source code bug and this starts to fail when it's not intended to fail, we will never know since the error is ignored.
2. If somehow deserialized incorrect data, we will be stuck with it forever (since we don't try to mutate it here), but it also won't be deserialized successfully in executor.

Thinking what's a better way to handle this...
We generally ensure correctness of programs during deserialization. I think we need to try to decompress BufferCompressed during deserialization and then either fail in strict mode, or replace it with an empty buffer in non-strict mode (not sure else we can do). Then here we can panic on non-nil err.